src/RoutingList/RoutingListContainer.js

```diff
@@ -0,0 +1,122 @@
+import {
+  useCallback,
+  useState,
+} from 'react';
+import { omit } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import {
+  useHistory,
+  useParams,
+} from 'react-router';
+
+import { useShowCallout } from '@folio/stripes-acq-components';
+import {
+  ConfirmationModal,
+  LoadingView,
+} from '@folio/stripes/components';
+
+import { UNIQUE_NAME_ERROR_CODE } from './constants';
+import {
+  useRoutingListById,
+  useRoutingListMutation,
+} from './hooks';
+import RoutingListForm from './RoutingListForm';
+
+import css from './RoutingList.css';
+
+export const RoutingListContainer = () => {
```

Comment:
If this component is a form container, please indicate this in the component name.